compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/RepoUtils.java

```diff
@@ -159,11 +159,21 @@ public static boolean validateOrg(String orgName) {
     /**
      * Validates the org-name and package name.
      *
-     * @param pkgName The org-name or package name.
-     * @return True if valid org-name or package name, else false.
+     * @param pkgName The package name.
+     * @return True if valid package name, else false.
      */
     public static boolean validatePkg(String pkgName) {
         String validRegex = "^[a-zA-Z0-9_.]*$";
         return Pattern.matches(validRegex, pkgName);
     }
+
+    /**
+     * Check if the org-name is a reserved org-name in ballerina.
+     *
+     * @param orgName The org-name
+     * @return True if the org-name is reserved, else false.
+     */
+    public static boolean isReservedOrgName(String orgName) {
+        return orgName.equals("ballerina") || orgName.equals("ballerinax");
```

Comment:
Shall we introduce two constants for "ballerina" and "ballerinax" ?

Reply:
Fixed with ff52b4f